packages/website/src/pages/Index.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <home-view :content="content" />
+</template>
+
+<script lang="ts">
+import { Component } from 'vue-property-decorator';
+
+import HomeView from '@/views/Home.vue';
+import Page from '@/page';
+
+import { HomeContent } from '@/content';
+
+import json from '../../data/pages/home.json';
+
+interface HomePage {
+  content: HomeContent;
+}
+
+@Component({
+  metaInfo: {
+    title: 'Home'
+  },
+  components: {
+    HomeView
+  }
+})
+class Home extends Page<HomeContent>(json) {}
```

Comment:
so higher-order, very functional

Reply:
I went back-and-forth on how to do this. Decorators were too messy and this easily works with Vue's reactivity so :+1: